cypress/e2e/integration/app-frontend/group.js

```diff
@@ -245,6 +245,29 @@ describe('Group', () => {
     cy.get(appFrontend.group.prefill.liten).click();
     cy.contains(mui.button, texts.next).click();
     expectRows(['NOK 9 872 345', 'NOK 18 872 345']);
+
+    cy.contains(mui.button, texts.prev).click();
+    cy.get(appFrontend.group.prefill.marked).click();
+    cy.contains(mui.button, texts.next).click();
+    expectRows(['NOK 9 872 345', 'NOK 18 872 345'], ['NOK 1 234', 'NOK 4 321']);
+    cy.get(appFrontend.group.mainGroup)
+      .find(mui.tableBody)
+      .then((table) => {
+        cy.get(table).children().eq(0).find(appFrontend.group.delete).should('be.visible');
+        cy.get(table)
+          .children()
+          .eq(0)
+          .find(appFrontend.group.edit)
+          .should('be.visible')
+          .should('have.text', texts.edit);
+        cy.get(table).children().eq(1).find(appFrontend.group.delete).should('not.exist');
+        cy.get(table)
+          .children()
+          .eq(1)
+          .find(appFrontend.group.edit)
+          .should('be.visible')
+          .should('have.text', texts.view);
```

Comment:
```suggestion
        cy.wrap(table).children().eq(0).find(appFrontend.group.delete).should('be.visible');
        cy.wrap(table)
          .children()
          .eq(0)
          .find(appFrontend.group.edit)
          .should('be.visible')
          .should('have.text', texts.edit);
        cy.wrap(table).children().eq(1).find(appFrontend.group.delete).should('not.exist');
        cy.wrap(table)
          .children()
          .eq(1)
          .find(appFrontend.group.edit)
          .should('be.visible')
          .should('have.text', texts.view);
```

This will make sure it works cleanly with typescript after #735 :partying_face:. It turns out we've been using cypress wrong all along; we were meant to use the `wrap` function inside a `then`, not `get`.